src/__tests__/incidents/list/ViewIncidents.test.tsx

```diff
@@ -59,17 +59,12 @@ const setup = (permissions: Permissions[]) => {
 
 it('should filter incidents by status=reported on first load ', () => {
   setup([Permissions.ViewIncidents])
-
-  expect(IncidentRepository.search).toHaveBeenCalled()
-  expect(IncidentRepository.search).toHaveBeenCalledWith({ status: IncidentFilter.reported })
+  expect(screen.getByRole('combobox')).toHaveValue('incidents.status.reported')
 })
 
 describe('layout', () => {
   it('should render a table with the incidents', async () => {
-    const { container } = setup([Permissions.ViewIncidents])
-
-    await waitFor(() => {
-      expect(container.querySelector('table')).toHaveTextContent(IncidentFilter.reported)
-    })
+    setup([Permissions.ViewIncidents])
+    expect(await screen.findByRole('table')).toHaveTextContent(IncidentFilter.reported)
```

Comment:
hm, can we make an assertion based on some value from `expectedIncidents` here?

Reply:
@Aprillion you can add or modify after merge if you think there's a better way 👍🏻